Sources/MuxPlayerSwift/ReverseProxyServer/ReverseProxyServer.swift

```diff
@@ -268,10 +271,15 @@ class ReverseProxyServer {
     private func setupCMAFSegmentHandler() {
         self.webServer.addHandler(
             forMethod: "GET",
-            pathRegex: "^/.*\\.m4s$",
+            pathRegex: "^/.*\\.(m4s|mp4)$", // TODO: For the shortform proposal: test assets crrently have mp4
```

Comment:
yeah `'m4s'|'m4v'|'m4a'|'mp4'` are all valid extensions... my S1 is just account for all of them if we're using extension-based matching, but you're right that Mux Video doesn't use all of them (might only use `'m4s'`? Can't recall)